pkg/investigations/chgm/chgm.go

```diff
@@ -55,6 +55,17 @@ func InvestigateTriggered(r *investigation.Resources) error {
 		return r.PdClient.SilenceAlertWithNote(res.string() + "Unrelated limited support reason present on cluster, silenced.")
 	}
 
+	longHibernation, hibernationErr := investigateHibernation(r.Cluster, r.OcmClient)
+	if hibernationErr != nil {
+		logging.Warn(fmt.Errorf("Could not check hibernation status of cluster: %w", err))
+	}
+	if longHibernation {
+		err = r.PdClient.AddNote(fmt.Sprintf("Cluster was hibernated more than %.0f days - investigate CSRs and kubelet certificates: see hhttps://github.com/openshift/ops-sop/blob/master/v4/alerts/cluster_has_gone_missing.md#24-hibernation", hibernationTooLong.Hours()/24))
```

Comment:
```suggestion
		err = r.PdClient.AddNote(fmt.Sprintf("⚠️ Cluster was hibernated more than %.0f days - investigate CSRs and kubelet certificates: see hhttps://github.com/openshift/ops-sop/blob/master/v4/alerts/cluster_has_gone_missing.md#24-hibernation", hibernationTooLong.Hours()/24))
```

Just so that this is easier to catch when looking at the pagerduty alert. Text without flashy emojis there tends to get overlooked, especially with the wall of text we post for CHGM.